src/qibolab/__init__.py

```diff
@@ -79,3 +79,39 @@ def execute_qasm(circuit: str, platform, runcard=None, initial_state=None, nshot
     return QibolabBackend(platform, runcard).execute_circuit(
         circuit, initial_state=initial_state, nshots=nshots
     )
+
+
+class MetaBackend:
+    """Meta-backend class which takes care of loading the qibolab backend."""
+
+    @staticmethod
+    def load(platform: str):
+        """Loads the backend.
+
+        Args:
+            platform (str): Name of the platform to load.
+        Returns:
+            qibo.backends.abstract.Backend: The loaded backend.
+        """
+        from qibolab.backends import QibolabBackend
+
+        PLATFORMS = os.environ.get(PLATFORMS)
```

Comment:
Maybe you want to have a dedicated function for this, but you could see it from above (L49): the `PLATFORMS` variable contains the directory where the platforms are located, not the list of platforms.

To generate the list, you need something like:
```suggestion
        platformsdir = Path(os.environ.get(PLATFORMS))
        platforms = [d.name for d in platforms.iterdir() if d.is_dir() and not d.name.startswith("_")]
```